File: Barking Dog/BarkingDog.swift
func bark(barking: Bool,hourOfTheDay:Int)->Bool{
    if (barking == true){
        if ((hourOfTheDay==23 || hourOfTheDay<8) && hourOfTheDay>=0){
            return true;
        }
    }
    return false;
}
print(bark(barking: true, hourOfTheDay: 3));

File: Barking Dog/BarkingDog.js
function bark(barking,houroftheday){
    if (barking == true){
        if ((houroftheday==23 || houroftheday<8) && houroftheday>=0)
            console.log( "true");
        else{
            console.log("false");
        }
    }else{
        console.log("false");
    }
}
bark(true,-1);
